app/driver.go

```diff
@@ -245,6 +245,21 @@ func deployCSIDriver(kubeClient *clientset.Clientset, lhClient *lhclientset.Clie
 		return err
 	}
 
+	// Longhorn #1365, remove compatible CSI deployment
```

Comment:
Add a `TODO: Remove This after v1.1.0`.

Since we shouldn't have any user use compatible CSI deployment after v1.1.0.

This reminds us to remove this part of the code later.